packages/api/src/task/scheduler.ts

```diff
@@ -18,6 +18,9 @@ const taskInfo = (task: Task): messages.TaskInfo => ({
   snapshot: task,
 });
 
+const MAX_RETRY_ATTEMPTS = 3;
```

Comment:
Just to align on the terms here... Attempt means a different thing than retries. If the task executes 4 times, it's 4 attempts but only 3 retries (but 4 "tries"). I already suggested dropping the "attempt" term and just using "retry" instead, and so we can simplify this const to just `MAX_RETRIES` as well.

Reply:
Makes much sense, updated